group_vars/all.yml

```diff
@@ -0,0 +1,56 @@
+cluster_name: "Yugabyte_cluster"
+type_instance: "t2.micro"
+# aws.networking
+vpc_name:       'Yugabyte_VPC_Res'
+vpc_cidr_2:     '185'
+vpc_cidr: '10.{{vpc_cidr_2}}.0.0/16'
+vpc_create: true
```

Comment:
would be good to add this to the readme, that you can skip the vpc_create by setting this flag to `false`

Reply:
I will add it asap.

Reply:
Change adjusted as per the new changes (mentioned in the documentation too).